notebooks/visualization/MetPy SkewT.ipynb

```diff
@@ -0,0 +1,298 @@
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Add a hodograph\n",
+    "\n",
+    "In this section, we'll add a hodograph inset to the plot.\n",
+    "Everything above the two commented lines is the same as the previous skew-T plot. The next two lines add a hodograph inset using matplotlib's inset_axes functionality and MetPy's Hodograph class.\n",
+    "\n",
+    "The next line sets the range of the hodograph (from + to - 80 kt in this case).\n",
+    "\n",
+    "Finally, the last two lines plot wind data below 150 hPa on the hodograph and color it by wind speed, and tell matplotlib to display the plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(9, 11))\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "#########################################################\n",
+    "#########################################################\n",
+    "\n",
+    "ax_hod = inset_axes(skew.ax, '30%', '30%', loc=1)\n",
+    "h = Hodograph(ax_hod, component_range=80)\n",
+    "\n",
+    "h.add_grid(increment=20)\n",
+    "\n",
+    "h.plot_colormapped(u[p>150*units('hPa')], v[p>150*units('hPa')], spd[p>150*units('hPa')])\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## See also\n",
+    "\n",
+    "Documentation for:\n",
+    "\n",
+    "* [metpy.plots.SkewT](https://unidata.github.io/MetPy/latest/api/generated/metpy.plots.SkewT.html#metpy.plots.SkewT)\n",
+    "* [metpy.plots.Hodograph](https://unidata.github.io/MetPy/latest/api/generated/metpy.plots.Hodograph.html#metpy.plots.Hodograph)\n",
+    "\n",
+    "## Other Notebooks\n",
+    "\n",
+    "* [A more detailed introduction to plotting Skew-Ts](https://unidata.github.io/python-training/workshop/Skew_T/skewt-and-hodograph/)\n",
```

Comment:
Could you also add a link to the python-awips Upper Air Obs noteboook:  https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/visualization/python-awips-WorkingWithUpperAirObs.ipynb

It also uses Metpy to create a SkewT and might be a nice introduction on how to access that kind of data using python-awips as an alternative.